CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# v0.0.17 (2021-06-09)

```

Comment:
Can you change the date for this wednesday. Let's publish a new release on that day.
```suggestion
# v0.0.17 (2021-06-30)

```